js/preload/preloadMain.js

```diff
@@ -367,8 +367,17 @@ function createAPI() {
         });
     }
 
+    // reset the badge count when ever user refresh
```

Comment:
Please change the comment to below
`// reset the badge count whenever an user refreshes the electron client`